leanpkg.toml

```diff
@@ -1,8 +1,9 @@
 [package]
 name = "polya"
 version = "0.1"
-lean_version = "3.3.0"
+lean_version = "3.4.2"
+path = "src"
 
 [dependencies]
-mathlib = {git = "https://github.com/leanprover/mathlib", rev = "lean-3.3.0"}
+mathlib = {path = "/home/pnm/leanprover/mathlib/"}

```

Comment:
Can you remove this change? The version update is good, and you can keep this local. (Maybe add `leanpkg.toml` to .gitignore.)